addons/hr_recruitment/hr_recruitment.py

```diff
@@ -540,12 +540,19 @@ def write(self, cr, uid, ids, vals, context=None):
 
     def _broadcast_welcome(self, cr, uid, employee_id, context=None):
         """ Broadcast the welcome message to all users in the employee company. """
+        model_data_pool = self.pool['ir.model.data']
+        template_pool = self.pool['mail.template']
```

Comment:
use camel case notation for model (such as `MailTemplate`and `IrModelData`). This will help the migration to new API ;)